utils.cpp

```diff
@@ -0,0 +1,58 @@
+ 
+#include <iostream>
+#include <string>
+#include <vector>
+
+using namespace std;
+// To make the system more flexible I prefer not to make a subclass for every person and room type
```

Comment:
in this small program - might be, but even here the main is not so readable... In the big system, it will be totally wrong way